mappings/net/minecraft/world/gen/chunk/placement/StructurePlacement.mapping

```diff
@@ -27,6 +27,8 @@ CLASS net/minecraft/class_6874 net/minecraft/world/gen/chunk/placement/Structure
 		ARG 5 frequency
 	METHOD method_41636 getLocatePos (Lnet/minecraft/class_1923;)Lnet/minecraft/class_2338;
 		ARG 1 chunkPos
+	METHOD method_41637 buildCodec (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/Products$P5;
+		ARG 0 instance
```

Comment:
Do we generally call this `instance` in Yarn? That name is a bit vague since it follows DFU's naming pattern of calling all typeclass implementation classes `Instance` and placing them inside the target class.

(I've sometimes called it `builder` in my own code but that isn't exactly accurate either since it works with RecordCodecBuilders and is not an RCB itself)

Reply:
Hmm, there isn't a good alternative, and it's already used a million times... maybe it's ok?

Reply:
Yep, seems fine enough.